apps/src/templates/projects/ProjectCardGrid.story.jsx

```diff
@@ -0,0 +1,484 @@
+      publishedToClass: false
+    },
+    currentGallery: "public"
+  },
+  {
+    projectData: {
+      channel: 'ABCDEFGHIJKLM01234',
+      projectName: 'Playlab 3',
+      studentName: 'Penelope',
+      studentAge: 13,
+      type: 'playlab',
+      updatedAt: '2016-12-31T23:59:59.999-08:00',
+      publishedToPublic: true,
+      publishedToClass: false
+    },
+    currentGallery: "public"
+  },
+  {
+    projectData: {
+      channel: 'ABCDEFGHIJKLM01234',
+      projectName: 'Playlab 4',
+      studentName: 'Penelope',
+      studentAge: 18,
+      type: 'playlab',
+      updatedAt: '2016-12-31T23:59:59.999-08:00',
+      publishedToPublic: true,
+      publishedToClass: false
+    },
+    currentGallery: "public"
+  },
+  {
+    projectData: {
+      channel: 'ABCDEFGHIJKLM01234',
+      projectName: 'Playlab 5',
+      studentName: 'Penelope',
+      studentAge: 14,
+      type: 'playlab',
+      updatedAt: '2016-12-31T23:59:59.999-08:00',
+      publishedToPublic: true,
+      publishedToClass: false
+    },
+    currentGallery: "public"
+  },
+];
+
+const PERSONAL_PROJECTS = [
+  {
+    projectData: {
+      channel: 'ABCDEFGHIJKLM01234',
+      projectName: 'Personal 1',
+      studentName: 'Penelope',
+      type: 'applab',
+      updatedAt: '2016-12-31T23:59:59.999-08:00',
+      publishedToPublic: false,
+      publishedToClass: false
+    },
+    currentGallery: "personal"
+  },
+  {
+    projectData: {
+      channel: 'ABCDEFGHIJKLM01234',
+      projectName: 'Personal 2',
+      studentName: 'Penelope',
+      type: 'applab',
+      updatedAt: '2016-11-30T23:59:59.999-08:00',
+      publishedToPublic: true,
+      publishedToClass: false
+    },
+    currentGallery: "personal"
+  },
+  {
+    projectData: {
+      channel: 'ABCDEFGHIJKLM01234',
+      projectName: 'Personal 3',
+      studentName: 'Penelope',
+      type: 'applab',
+      updatedAt: '2016-10-31T23:59:59.999-08:00',
+      publishedToPublic: false,
+      publishedToClass: true
+    },
+    currentGallery: "personal"
+  },
+  {
+    projectData: {
+      channel: 'ABCDEFGHIJKLM01234',
+      projectName: 'Personal 4',
+      studentName: 'Penelope',
+      type: 'applab',
+      updatedAt: '2016-10-30T23:59:59.999-08:00',
+      publishedToPublic: true,
+      publishedToClass: true
+    },
+    currentGallery: "personal"
+  },
+  {
+    projectData: {
+      channel: 'ABCDEFGHIJKLM01234',
+      projectName: 'Personal 5',
+      studentName: 'Penelope',
+      type: 'applab',
+      updatedAt: '2016-10-29T23:59:59.999-08:00',
+      publishedToPublic: false,
+      publishedToClass: false
+    },
+    currentGallery: "personal"
+  },
+  {
+    projectData: {
+      channel: 'ABCDEFGHIJKLM01234',
+      projectName: 'Personal 6',
+      studentName: 'Penelope',
+      type: 'applab',
+      updatedAt: '2016-10-28T23:59:59.999-08:00',
+      publishedToPublic: false,
+      publishedToClass: false
+    },
+    currentGallery: "personal"
```

Comment:
You might find it useful to create some helper methods to generate your test data. Maybe something like `generateFakeProject`. You could even combine with the approach from my other comment, where you perhaps have a default set of data, and then generateFakeProject takes a param of fields to override. i.e. you'd have something like:
```
function generateFakeProject(overrideData) {
  return {
    ...defaultProject,
    ...overrideData
  };
}

// elsewhere
generateFakeProject({publishedToClass: true});
```